.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+##### Thanks for considering contributing to HTTPX!
+
+Our [documentation on contributing to HTTPX]() contains information on
+how to report bugs, write and test new features, and debug issues with
+your own changes.
```

Comment:
Could this file be located in the project root directory? That's where most people will look for first I believe, and that location is also correctly recognized by GitHub in community insights etc.

Reply:
Github also recognizes this directory for contributing.md, code_of_conduct, etc.  We already have a lot top-level and I'm planning on adding an issue_template.md eventually as well.

Reply:
Sure, just saying that a CONTRIBUTING file at the root level is immediately discoverable, be it by repo visitors or by a developer looking at their file explorer tree — no need to open a folder to see it.

Reply:
Sure, I'll move it to the root.